packages/eslint-config-airbnb-base/README.md

```diff
@@ -0,0 +1,33 @@
+# eslint-config-airbnb-base
+
+[![npm version](https://badge.fury.io/js/eslint-config-airbnb-base.svg)](http://badge.fury.io/js/eslint-config-airbnb-base)
+
+This package provides Airbnb's base JS .eslintrc as an extensible shared config.
```

Comment:
Might it make sense to add a note here to point people somewhere else if they want React rules as well?